kgforge/specializations/resolvers/demo_resolver.py

```diff
@@ -63,7 +59,7 @@ def _resolve(self, text: Union[str, List[str]], target: Optional[str], type: Opt
                 )
             except StopIteration:
                 return None
-        elif strategy == ResolvingStrategy.EXACT_CASEINSENSITIVE_MATCH:
+        elif strategy == ResolvingStrategy.EXACT_CASE_INSENSITIVE_MATCH:
```

Comment:
if you are changing this, you should also update the notebook in use-cases about resolving

Reply:
It's in both, I just pushed the change